courses/free-courses-en.md

```diff
@@ -523,6 +523,7 @@
 * [Deep Learning With Tensorflow 2.0 and Keras](https://youtube.com/playlist?list=PLeo1K3hjS3uu7CxAacxVndI4bE_o3BDtO) - codebasics
 * [Deep Multi-Task and Meta Learning](https://cs330.stanford.edu) - Chelsea Finn (Stanford University)
 * [Deep Reinforcement Learning](http://rail.eecs.berkeley.edu/deeprlcourse/) - Sergey Levine
+* [Deep Reinforcement Learning Course - Hugging Face](https://huggingface.co/learn/deep-rl-course/unit0/introduction)
```

Comment:
```suggestion
* [Deep Reinforcement Learning Course](https://huggingface.co/learn/deep-rl-course/unit0/introduction) (Hugging Face)
```

Reply:
when you've made changes, please add `(changes made)` to the title of your PR

Reply:
> when you've made changes, please add `(changes made)` to the title of your PR
Changed the PR title.